File: lab4/python/main.py
import datetime as dt

a = 0
b = 2
c = 3
iterations = 100000000

start = dt.datetime.now()

for i in range(iterations):
    a += b * 2 + c - i

end = dt.datetime.now()

exec_time = ((end - start).seconds * 1000000 + (end-start).microseconds) / 1000000
print(f'Python: {exec_time} seconds')

File: lab4/java/Main.java
public class Main {

    public static void main(String[] args) {
        int a = 0;
        int b = 2;
        int c = 3;
        long iterations = 100000000l;
        long start = System.currentTimeMillis();
        long end;
        float exec_time;

        for (int i = 0; i <= iterations; i++)
            a += b * 2 + c - i;

        end = System.currentTimeMillis();

        exec_time = (float)(end - start) / 1000;

        System.out.println("Java: " + exec_time + " seconds");
    }
}
